### PR TITLE
Ch 2975: Change the way variations/goals/audiences are added within UX

### PR DESCRIPTION
**Pull request by eshtadc**
Most of the work is to support the fact that now there can be multiple new items per form.

### CODE REVIEW

File: acquia_lift.admin.wizard.inc [hunk 1]
```diff
@@ -196,139 +196,153 @@ function acquia_lift_personalize_campaign_wizard_variations_alter(&$form, &$form
...
+          '#ajax' => array(
+            'callback' => 'acquia_lift_personalize_campaign_wizard_variations_ajax',
+            'wrapper' => $new_option_sets_id,
           ),
         );
-        $form['variations']['add_variation']['details']['element']['content'] = _acquia_lift_personalize_campaign_wizard_variations_element($form, $form_state, $agent_data);
-        break;
+      }
+      else {
+        // Display a option set type specific form for the new goal.
```
Comment (katbailey) [inline acquia_lift.admin.wizard.inc:260]: New goal? Is this a copy/paste error?

File: acquia_lift.admin.wizard.inc [hunk 2]
```diff
@@ -196,139 +196,153 @@ function acquia_lift_personalize_campaign_wizard_variations_alter(&$form, &$form
...
+                'wrapper' => $new_option_sets_id,
+              ),
+            );
+            break;
+        }
+      }
     }
   }
-
+  // The button to add a new goal.
```
Comment (katbailey) [inline acquia_lift.admin.wizard.inc:333]: Moar copy/paste? :)
I'm really catching good stuff here...

File: acquia_lift.admin.wizard.inc (hunk 1 above)
Comment (eshtadc) [inline acquia_lift.admin.wizard.inc:260]: Seems to be a half-hearted copy/pasta attempt.

File: acquia_lift.admin.wizard.inc (hunk 2 above)
Comment (eshtadc) [inline acquia_lift.admin.wizard.inc:333]: Fixed.

**Pull request closed without merging**